roles/openvpn/tasks/main.yml

```diff
@@ -82,19 +80,16 @@
     - { src: 'silence', dest: '/etc/openvpn/scripts/silence', mode: '0755' }
     - { src: 'xscenet.conf.j2', dest: '/etc/openvpn/xscenet.conf', mode: '0644' }
     - { src: 'openvpn_handle.j2', dest: '/etc/iiab/openvpn_handle', mode: '0644' }
-    - { src: 'iiab-support', dest: '/usr/bin/iiab-support', mode: '0755' }
     - { src: 'iiab-remote-on.j2', dest: '/usr/bin/iiab-remote-on', mode: '0755' }
     - { src: 'iiab-remote-off', dest: '/usr/bin/iiab-remote-off', mode: '0755' }
-    # Comment out in future?  Not recommended as of August 2018:
-    - { src: 'iiab-handle.j2', dest: '/usr/bin/iiab-handle', mode: '0755' }
-    # Obsolete & unused for ~2 years as of August 2018:
-    # - { src: 'up_wan', dest: '/usr/lib/iiab/up_wan', mode: '0755' }
-    # Obsolete & unused for ~2 years as of August 2018:
-    #- { src: 'start.j2', dest: '/usr/lib/iiab/start', mode: '0755' }
-    # Obsolete & unused for ~2 years as of August 2018:
-    #- { src: 'iiab-vpn.conf.in', dest: '/etc/openvpn/iiab-vpn.conf.in', mode: '0644' }
-    # Obsolete & unused for ~2 years as of August 2018:
-    #- { src: 'iiab-vpn.j2', dest: '/usr/bin/iiab-vpn', mode: '0755' }
+    # 2019-10-09: Not recommended for over a year & no longer in use
+    # - { src: 'iiab-handle.j2', dest: '/usr/bin/iiab-handle', mode: '0755' }
+
+- name: Copy /opt/iiab/iiab/iiab-support to /usr/bin/iiab-support, in case git tree deleted e.g. on a smaller IIAB install
+  copy:
+    src: "{{ iiab_dir }}/iiab-support"
+    dest: /usr/bin/
+    mode: 0755
 
```

Comment:
if the git tree is deleted, then none of the ansible routines are available

Reply:
True.  If there are any especially important scripts that need to elevated from `/opt/iiab/iiab/scripts` or `/opt/iiab/iiab` to `/usr/bin` LMK and I will do that.

But we might possibly be ok as is?

Reply:
Think I see what you want with /usr/bin/iiab-support post install but the log directory must exist, a gratuitous "mkdir -p" in place of "cd" can't hurt.

Reply:
I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Reply:
I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Reply:
I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Reply:
Ok.  See https://github.com/holta/iiab/blob/support_yml/iiab-support#L59-L70

All code paths tested.

Should be ready to merge.